src/DependencyInjection/Configuration/Configuration.php

```diff
@@ -119,6 +119,7 @@ private function addClientsSection(ArrayNodeDefinition $rootNode): void
                                     ->booleanNode('connection_async')->defaultFalse()->end()
                                     ->booleanNode('connection_persistent')->defaultFalse()->end()
                                     ->scalarNode('connection_timeout')->cannotBeEmpty()->defaultValue(5)->end()
+                                    ->variableNode('ssl_context')->defaultNull()->end()
```

Comment:
This should go under `parameters`, like all other connection related settings.

Reply:
@ostrolucky ok, I'll move this option under `paramenters`